### DISCUSSION TITLE
umi max 微前端的MicroApp组件，在有routes配置时正常，但是在约定式路由时，打开有子应用的页面，页面闪一下后报错

### DISCUSSION THREAD

**Discussion by ayahua** [umijs/umi#8869] [Q&A]
umi max 微前端的MicroApp组件，在有routes配置时正常，但是在约定式路由时，打开有子应用的页面，页面闪一下后报错，提示匹配不到对应路由。
![image](https://user-images.githubusercontent.com/36917157/182809229-0f7396de-aad0-4939-981e-9d0c74af4815.png)

**Comment by fz6m** [comment]
由于缺乏足够的信息，请提供 [最小重现](https://stackoverflow.com/help/minimal-reproducible-example) 的 demo 帮助排查问题。

**Comment by giscafer** [comment]
same issue
